dScripts/CppScripts.cpp

```diff
@@ -327,668 +327,382 @@
-
-	// Alpha
-	if (scriptName == "scripts\\ai\\FV\\L_TRIGGER_GAS.lua")
-		script = new TriggerGas();
-	else if (scriptName == "scripts\\ai\\FV\\L_ACT_NINJA_SENSEI.lua")
-		script = new ActNinjaSensei();
-
-	// pickups
-	if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_1_BRONZE-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(1);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_1_GOLD-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(10000);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_1_SILVER-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(100);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_10_BRONZE-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(10);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_10_GOLD-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(100000);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_10_SILVER-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(1000);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_25_BRONZE-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(25);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_25_GOLD-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(250000);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_25_SILVER-COIN-SPAWNER.lua")
-		script = new SpecialCoinSpawner(2500);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_IMAGINE-POWERUP-SPAWNER.lua")
-		script = new SpecialPowerupSpawner(13);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_IMAGINE-POWERUP-SPAWNER-2PT.lua")
-		script = new SpecialPowerupSpawner(129);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_LIFE-POWERUP-SPAWNER.lua")
-		script = new SpecialPowerupSpawner(5);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_ARMOR-POWERUP-SPAWNER.lua")
-		script = new SpecialPowerupSpawner(747);
-	else if (scriptName == "scripts\\ai\\SPEC\\L_SPECIAL_SPEED_BUFF_SPAWNER.lua")
-		script = new SpecialSpeedBuffSpawner();
-
-	// Wild
-	if (scriptName == "scripts\\ai\\WILD\\L_WILD_GF_RAT.lua" || scriptName == "scripts\\ai\\WILD\\L_WILD_GF_SNAIL.lua")
-		script = new WildAndScared();
-	else if (scriptName == "scripts\\ai\\WILD\\L_WILD_GF_GLOWBUG.lua")
-		script = new WildGfGlowbug();
-	else if (scriptName == "scripts\\ai\\WILD\\L_WILD_AMBIENT_CRAB.lua")
-		script = new WildAmbientCrab();
-	else if (scriptName == "scripts\\ai\\WILD\\L_WILD_PANTS.lua")
-		script = new WildPants();
-	else if (scriptName == "scripts\\ai\\WILD\\L_WILD_NINJA_BRICKS.lua")
-		script = new WildNinjaBricks();
-	else if (scriptName == "scripts\\ai\\WILD\\L_WILD_NINJA_STUDENT.lua")
-		script = new WildNinjaStudent();
-	else if (scriptName == "scripts\\ai\\WILD\\L_WILD_NINJA_SENSEI.lua")
-		script = new WildNinjaSensei();
-	else if (scriptName == "scripts\\ai\\WILD\\L_LUP_generic_interact.lua")
-		script = new LupGenericInteract();
-	else if (scriptName.rfind("scripts\\zone\\LUPs\\RobotCity Intro\\WBL_RCIntro_RobotCitizen", 0) == 0)
```

Comment:
Where did these go?

Reply:
there should be 4-5 with this style name we need to load, its the only missing script from the transfer

Reply:
There are 5, one for each color. Currently at lines 680-684 in the CPPScripts.cpp file

If this is still a question let me know